config/application.rb

```diff
@@ -12,7 +12,7 @@
 require "action_text/engine"
 require "action_view/railtie"
 require "action_cable/engine"
-# require "sprockets/railtie"
+require "sprockets/railtie"
```

Comment:
Uncommented this because of the same tutorial mentioned above re: graphiql errors.